scripts/solve.js

```diff
@@ -1,7 +1,14 @@
+const sampleCode = 'function myScript(){return 100;}\n';
 window.addEventListener('load', function() {
   var myCodeMirror = CodeMirror(document.getElementById('code-area'), {
-    value: "function myScript(){return 100;}\n",
-    mode:  "javascript",
+    value: sampleCode,
+    mode: 'javascript',
     lineNumbers: true
   });
+
+  JSHINT(sampleCode);
```

Comment:
Maybe refactor this into a function?
That way we can easily move this functionality to an event listener to implement live analysis.

Reply:
good idea, done!